app/models/form_profiles/va_2122a.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FormProfiles::VA2122a < FormProfile
+  def metadata
+    {
+      version: 0,
+      prefill: true,
+      returnUrl: '/claimant-personal-information'
```

Comment:
How have you tested that `'/claimant-personal-information'` is valid?

Reply:
Hi @rmtolmach - is the `returnUrl` being tested for any of the other form profiles? I didn't immediately find any examples of that.

Reply:
@cosu419 According to the docs, in this issue, the returnUrl is '/claimant-information'

Reply:
<img width="531" alt="Screenshot 2024-08-09 at 3 26 33 PM" src="https://github.com/user-attachments/assets/bc439eeb-de3d-40db-96a0-a604bfb04eb1">

Reply:
I also see `/claimant-contact-information`. Can you confirm this is a valid url?

Reply:
It looks like `/claimant-information` would be appropriate here. Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/17598/commits/c74b7a0a283e796f8e0a3604d99988d86958a343).